src/System.Private.ServiceModel/tests/Scenarios/Client/ExpectedExceptions/ExpectedExceptionTests.cs

```diff
@@ -278,7 +278,7 @@ public static void DuplexCallback_Throws_FaultException_ReturnsFaultedTask()
     }
 
     [Fact]
-#if FEATURE_NETNATIVE
+#if FULLXUNIT_NOTSUPPORTED
     [ActiveIssue(833)] // Not supported in NET Native
```

Comment:
Yeah, I was going to do that separately, as soon as I merge this PR I will validate those tests.